File: src/main/java/com/lannstark/lec08/Lec08Main.java
package com.lannstark.lec08;

public class Lec08Main {

  public static void main(String[] args) {

  }

  public int max(int a, int b) {
    if (a > b) {
      return a;
    }
    return b;
  }

  public static void printAll(String... strings) {
    for (String str : strings) {
      System.out.println(str);
    }
  }

}

File: src/main/kotlin/com/lannstark/lec08/lec08Main.kt
package com.lannstark.lec08

fun main() {
//    repeat("Hello World", 3, false)
    repeat("Hello World", useNewLine = false)
}

fun repeat(
    str: String,
    num: Int = 3,
    useNewLine: Boolean = true
) {
    for (i in 1..num) {
        if (useNewLine) {
            println(str)
        } else {
            println(str)
        }
    }
}

fun max(a: Int, b: Int): Int =
    if (a > b) {
        a
    } else {
        b
}

fun max2(a: Int, b: Int) = if (a > b) a else b

